codechecker_common/output/twodim.py

```diff
@@ -108,6 +108,11 @@ def __to_table(lines, separate_head=True, separate_footer=False):
     # Print the actual data.
     str_parts.append("-" * (sum(widths) + 3 * (len(widths) - 1)))
     for i, line in enumerate(lines):
+        # It is possible that one of the item in the line is None which will
+        # raise an exception when passed to the format function below. For
+        # this reason we will convert None to an empty string here.
+        line = [ln if ln is not None else '' for ln in line]
```

Comment:
Column width computation also assumes that a line consists of strings.So the line contents should be stringified at the beginning of this function instead:
```.py
lines = map(lambda line: list(map(str, line)), lines)
```
Or something similar :)

Reply:
Yes, nice catch! I created a helper function in this file to convert None values to valid strings. I also use this function when we print lines in csv or rows format.

Reply:
`old_val` parameter is always `None`, so it's not necessary. I wouldn't even use `-` in case of CSV, because this is a format for script (i.e. non-human) processing. In this case you could simply write `[[str(e) for e in line] for line in lines]`. This doesn't even require a separate function, you can assign this expression to `lines`.

Reply:
@bruntib Okay, I have updated this PR.